source/io_netcdf.F90

```diff
@@ -2070,26 +2062,9 @@ subroutine write_nstd_netcdf(data_file,field_id,            &
       write_error,       &! error flag
       supported
 
-   real (r4), allocatable, dimension (:,:,:,:,:)  ::  &
-      outdata_5d_r4
-
-   real (r4), allocatable, dimension (:,:,:,:)    ::  &
-      outdata_4d_r4
-
-   real (r4), allocatable, dimension (:,:,:)      ::  &
-      outdata_3d_r4
-
-   real (r4), allocatable, dimension (:,:)        ::  &
-      outdata_2d_r4
-
-   real (r8), allocatable, dimension (:)          ::  &
-      outdata_1d_r8
```

Comment:
I like removing `outdata_1d_r8`, and it looks like you can get rid of `outdata_2d_r8` as well.

Reply:
ugh, I was expecting the preview box to show lines below my selected line rather than above... but [L2065-L2067](https://github.com/ESCOMP/POP2-CESM/blob/954c0c5d43648ef8a60243b245cfb9f5cc479516/source/io_netcdf.F90#L2065) of `io_netcdf.F90` in your branch are not needed.